statvar_imports/ndap/nss_education/README.md

```diff
@@ -0,0 +1,17 @@
+source
+: https://ndap.niti.gov.in/catalogue?query=Household+Social+Consumption+-+Education&search=Variables%2CDatasetInfo&domain=ndap
+How to download
+
+type of place: Country/State
+
+Place Resolution
+Place Resolution if any
+### How to run:
+
+python3 stat_var_processor.py --input_data='statvar_imports/ndap/nss_education/test_data/sample_input/Proportion ( per 1000 ) of never enrolled persons (age 5- 29 years) and their per 1000 distribution - data.csv'   --pv_map='statvar_imports/ndap/nss_education/pvmap/pv_map.csv'  --config='statvar_imports/ndap/nss_education/pvmap/Proportion ( per 1000 ) of never enrolled persons (age 5- 29 years) and their per 1000 distribution_metadata.csv'  --output_path=statvar_imports/ndap/nss_education/test_data/sample_output/Proportion ( per 1000 ) of never enrolled persons (age 5- 29 years) and their per 1000 distribution
```

Comment:
This has a single pvmap invocation. 
Pls include all invocations for all metadata/pvmaps in the PR.